databases/week3/Additional_queries.sql

```diff
@@ -0,0 +1,43 @@
+-- Get meals that has a price smaller than a specific price fx 90
+
+select * from meal where price < 90;
+-- Get meals that partially match a title
+select * from meal where title like '%butter%'; 
+-- Get meals that still has available reservations
+SELECT meal.id as meal_id, title, discription, location, price, max_reservations, 
+	IFNULL(SUM(number_of_guests), 0) AS total_reservations, 
+    IFNULL((max_reservations - SUM(number_of_guests)), max_reservations) as available_spots
+FROM meal
+LEFT JOIN reservation
+	ON meal.id = meal_id
+GROUP BY meal.id
+HAVING  max_reservations > total_reservations or total_reservations is null
+order by available_spots;
```

Comment:
Consider using the COALESCE() function instead of IFNULL() : https://dev.mysql.com/doc/refman/8.0/en/comparison-operators.html#function_coalesce

Also you don't need the `or total_reservations is null` since you are already converting NULLs to zeroes